pkg/routeagent/controllers/route/route.go

```diff
@@ -67,6 +67,7 @@ const (
 	VxInterfaceWorker  = 0
 	VxInterfaceGateway = 1
 	VxLANPort          = 4800
+	VxLANOverhead      = 50
```

Comment:
So I assume this is subtracted from the system's default MTU as estimated overhead added by vxLAN stuff. But why 50? Should we make this configurable?

Reply:
50 is the VxLAN header size which is fixed.

Reply:
Yes, its the VxLAN header size which is fixed. I don't see a need for making it configurable.

Reply:
ok - makes sense.